test/COWFeeModule.t.sol

```diff
@@ -41,15 +45,19 @@ contract COWFeeModuleTest is Test {
         ISafe(targetSafe).enableModule(address(module));
     }
 
-    function testAuth() external {
```

Comment:
I think unit tests should aim to test one thing only. I broke this test in 3 unit tests